doc/building.html

```diff
@@ -395,6 +395,21 @@ <h3 id="windows">Windows</h3>
 to an outdated bash; While OpenJDK can be built with MSYS2, support for
 it is still experimental, so build failures and unusual errors are not
 uncommon.)</p>
+<p>Building and testing the JDK requires a well-defined locale to be
+guaranteed to run correctly. On non-Windows operating systems, this is
+achieved using the <code>LC_*</code> variables, which propagate to all
+child processes of the build. Unfortunately, there is no way to set the
+locale for a specific process like this in Windows. The only way is to
+change the user locale setting to <strong>US English</strong>, which
+will affect all applications run by the user.</p>
```

Comment:
IIUC, it is not only the user locale, but also the system locale that derives the file path encoding should be set to US English (using the `UTF-8` beta option may help though)

Reply:
Oh, here are lots of news to me.

1) What is the system locale? How do you check which locale is the system locale (as opposed to the user locale), and how to you change it?

2) What is the UTF-8 beta option?

Reply:
IMHO, Windows has a history of patching up locale support. They have at least System/User/UI default locales.

That said,
1) System locale is the locale of the OS that affects all users/non-Unicode ANSI applications. You can set it in the ControlPanel->Regional Settings->Administrative->"Change System Locale" button.

2) There is a check box that reads "Beta: Use Unicode UTF-8 for worldwide language support". This enables ANSI applications to use UTF-8 for wide char <-> multibyte conversion. 

So in a nutshell, these combinations of settings end up in many unexpected situations. YMMV